matchbox_socket/src/webrtc_socket/mod.rs

```diff
@@ -198,13 +205,15 @@ async fn message_loop<M: Messenger>(
 
             handshake_result = handshakes.select_next_some() => {
                 data_channels.insert(handshake_result.peer_id, handshake_result.data_channels);
-                peer_state_tx.unbounded_send((handshake_result.peer_id, PeerState::Connected)).expect("failed to report peer as connected");
+                peer_state_tx.unbounded_send((handshake_result.peer_id, PeerState::Connected))
+                    .map_err(TrySendError::into_send_error)?;
```

Comment:
ditto

Reply:
Yes, so propagating the error up seems reasonable.